src/commands/rp.ts

```diff
@@ -972,6 +972,49 @@ WHERE bank->>'${item.id}' IS NOT NULL;`);
 				});
 				return msg.channel.send('Globally synced slash commands.');
 			}
+			case 'migratenex': {
+				const nexItems = [
+					['Torva full helm (broken)', 'Torva full helm'],
+					['Torva platebody (broken)', 'Torva platebody'],
+					['Torva platelegs (broken)', 'Torva platelegs'],
+					['Torva boots (broken)', 'Torva boots'],
+					['Torva gloves (broken)', 'Torva gloves'],
+					['Pernix cowl (broken)', 'Pernix cowl'],
+					['Pernix body (broken)', 'Pernix body'],
+					['Pernix chaps (broken)', 'Pernix chaps'],
+					['Pernix boots (broken)', 'Pernix boots'],
+					['Pernix gloves (broken)', 'Pernix gloves'],
+					['Virtus mask (broken)', 'Virtus mask'],
+					['Virtus robe top (broken)', 'Virtus robe top'],
+					['Virtus robe legs (broken)', 'Virtus robe legs'],
+					['Virtus boots (broken)', 'Virtus boots'],
+					['Virtus gloves (broken)', 'Virtus gloves']
+				];
+				const res: any[] = await prisma.$queryRawUnsafe(`SELECT id
+FROM users
+WHERE "collectionLogBank"::jsonb ?| array['601', '605', '4272', '758', '759', '432', '709', '2404', '2838', '4273', '788', '983', '2409', '9654', '2423'];`);
+				let IDs = res.map(i => i.id);
+
+				for (const id of IDs) {
+					const user = await client.fetchUser(id).catch(noOp);
+					if (!user) continue;
+					const itemsToAddToTheirCL = new Bank();
+					const cl = user.cl();
+					if (!cl.has('Virtus crystal') && (cl.has('Virtus wand') || cl.has('Virtus book'))) {
```

Comment:
If you're giving people full credit for both books + wands, why not give them credit if they happen to roll a fresh Virtus crystal since the update? I would probably remove the `!cl.has('Virtus crystal)` but you might have a good reason it's there ?  
Maybe just subtract the existing number of crystals from the total of old ones. (To account for potentially created orbs/books after the update) (and checking it's > 0)